storage/container_storage_interface/persistent-storage-csi-vsphere.adoc

```diff
@@ -16,7 +16,7 @@ To create CSI-provisioned persistent volumes (PVs) that mount to vSphere storage
 
 * *vSphere CSI Driver Operator*: The Operator provides a storage class, called `thin-csi`, that you can use to create persistent volumes claims (PVCs). The vSphere CSI Driver Operator supports dynamic volume provisioning by allowing storage volumes to be created on-demand, eliminating the need for cluster administrators to pre-provision storage.
 
-* *vSphere CSI driver*: The driver enables you to create and mount vSphere PVs. In {product-title} 4.12, the driver version is 2.6.1.
+* *vSphere CSI driver*: The driver enables you to create and mount vSphere PVs. In {product-title} 4.12, the driver version is 2.6.1. The vSphere CSI driver supports the XFS file system.
```

Comment:
I think we should defer to RHEL docs. It is bit weird that we are documenting file system support, since it comes from OS. So something like:

"The vSphere CSI driver supports all the file systems supported by underlying Red Hat Core OS release, including XFS and Ext4.  Please refer to RHEL docs for supported file system - https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_file_systems/assembly_overview-of-available-file-systems_managing-file-systems (not sure if this is the right link though)"